<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thrid_window</class>
 <widget class="QWidget" name="thrid_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Snake_Robot_Controller</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/gui_png/rodu.png</normaloff>:/images/gui_png/rodu.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #333;</string>
  </property>
  <widget class="QDial" name="dial6">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>66</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>152</y>
     <width>129</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>152</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 7</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>143</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd9">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>132</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>216</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 8</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial7">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>132</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>216</y>
     <width>129</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 3</string>
   </property>
  </widget>
  <widget class="QDial" name="dial1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>66</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>86</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 1</string>
   </property>
  </widget>
  <widget class="QDial" name="dial9">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>256</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>336</y>
     <width>129</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>276</y>
     <width>129</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 4</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>143</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>256</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial10">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>316</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>86</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label10">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>336</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 10</string>
   </property>
  </widget>
  <widget class="QDial" name="dial8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>196</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd10">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>139</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>276</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Motor 9</string>
   </property>
  </widget>
  <widget class="QDial" name="dial5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>316</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>196</y>
     <width>60</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_connect">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>14</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/gui_png/home.png</normaloff>:/images/gui_png/home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>554</x>
     <y>386</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
font-family: Noto Sans;
font-style: normal;
font-weight: 600;
line-height: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/gui_png/move.png</normaloff>:/images/gui_png/move.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>70</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>thrid_window</receiver>
   <slot>page1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>thrid_window</receiver>
   <slot>send()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rad_up()</slot>
  <slot>slot1()</slot>
  <slot>rad_down()</slot>
  <slot>vel_up()</slot>
  <slot>vel_down()</slot>
  <slot>set_1()</slot>
  <slot>set_2()</slot>
  <slot>set_3()</slot>
  <slot>set_4()</slot>
  <slot>save()</slot>
  <slot>change()</slot>
  <slot>page1()</slot>
  <slot>send()</slot>
  <slot>slot3()</slot>
 </slots>
</ui>
